client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -149,34 +215,62 @@ const CancelPurchaseRefundInformation = ( {
 								checked={ cancelBundledDomain }
 								onChange={ onCancelBundledDomainChange }
 								label={
-									<>
-										{ i18n.translate( 'Cancel the plan {{em}}and{{/em}} the domain "%(domain)s."', {
-											args: {
-												domain: includedDomainPurchase.meta,
-											},
-											components: {
-												em: <em />,
-											},
-										} ) }
-										<br />
-										{ i18n.translate(
-											"You'll receive a full refund of %(planCost)s. The domain will be cancelled, and it's possible " +
-												"you'll lose it permanently.",
-											{
-												args: {
-													planCost: planCostText,
-												},
-											}
-										) }
-									</>
+									<Fragment>
+										<span className="cancel-bundled-domain__option-header">
+											{ translate(
+												'Cancel the plan {{strong}}and{{/strong}} the domain “%(domain)s”.{{br/}}',
+												{
+													args: {
+														domain: includedDomainPurchase.meta,
+													},
+													components: {
+														br: <br />,
+														strong: <strong />,
+													},
+												}
+											) }
+										</span>
+										<span className="cancel-bundled-domain__option-description">
+											{ translate(
+												"You'll receive a full refund of %(planCost)s. The domain will be cancelled, you risk losing it forever, and visitors to your site may experience difficulty accessing it.",
+												{
+													args: {
+														planCost: planCostText,
+													},
+												}
+											) }
+										</span>
+									</Fragment>
 								}
 							/>
-						</FormLabel>
+						</FormLabel>,
+						translate(
+							'Since you are cancelling your plan within %(refundPeriodInDays)d day of purchasing,',
+							'Since you are cancelling your plan within %(refundPeriodInDays)d days of purchasing,',
+							{
+								count: refundPeriodInDays,
+								args: {
+									refundPeriodInDays: refundPeriodInDays,
+								},
+							}
+						) +
+							' ' +
+							translate(
+								"you'll receive a {{spanRefund}}refund of %(refundAmount)s{{/spanRefund}} and it will be removed from your site immediately.",
```

Comment:
I saw the following on my screen here:

>Since you are cancelling your plan within 14 days of purchasing, [object Object]

I also don't think that translatable strings should be concatenated mid-sentence like this.  Shouldn't the whole sentence be translated as a single string?

Reply:
I replicated and fixed it.
<img width="1045" alt="Screenshot 2022-12-15 at 12 10 25" src="https://user-images.githubusercontent.com/5706607/207844704-cb8b021d-45a2-4aa9-9590-e24f87320ab3.png">